tbot/tbot.py

```diff
@@ -0,0 +1,115 @@
+# пример кода взят https://habr.com/ru/post/580408/
+import logger_config
+import project_settings
+import telebot
+import datetime
+import os, sys
+import logging
+import logging.config
+
+sys.path.insert(1, project_settings.PROJECT_PATH)
+
+import tbot.tbot_privatedata as tbot_privatedata
+import moysklad.moysklad_class_lib as ms_class_lib
+import googledrive.googledrive_class_lib as gs_class_lib
+import googledrive.googlesheets_vars as gs_vars
+import utils.file_utils
+
+
+logging.config.dictConfig(logger_config.LOGGING_CONF)
+logger = logging.getLogger("tbot")
+
+bot = telebot.TeleBot(tbot_privatedata.TOKEN,  parse_mode = None)
+
+
+@bot.message_handler(commands=['egais'])
+def start_message(message):
+    send_file = ''
+    unsuccess = False
+    try:
+        logger.debug("Приняли команду: " + message.json['text'])
+        bot.send_message(message.chat.id, "Готовлю данные...")
+
+        ms = ms_class_lib.MoySklad()
+        if not (ms is None):
```

Comment:
```suggestion
        if ms is not None:
```

При этом, ms никогда не None

Reply:
ok